astroquery/esasky/core.py

```diff
@@ -1523,6 +1547,13 @@ def _get_maps_for_mission(self, maps_table, mission, download_dir, cache, json,
 
         return maps
 
+    def _open_fits(self, path, verbose=False):
+        if verbose:
+            return fits.open(path)
+        with warnings.catch_warnings():
+            warnings.simplefilter("ignore")
```

Comment:
Maybe... Since ESASky exposes FITS files from so many missions, I feel that we still need a quite broad filter. I did some quick searches in `astropy.io.fits` and it seems that ignoring `VerifyWarning` might be better suited for this use case. That would let through `AstropyDeprecationWarning`, `AstropyUserWarning`, and potentially others.

Reply:
Yes, start with something small, and we can always add other specific ones. Otherwise, we indeed would miss the important and relevant ones, like deprecations, etc.